app/waf/src/models/adc.model.ts

```diff
@@ -125,7 +125,7 @@ export class Adc extends CommonEntity {
       response: true,
     },
   })
-  management: ConfigTypes['management'];
+  management?: ConfigTypes['management'];
```

Comment:
Is that posible to remove '?' ?

Reply:
This relates to the upper comments. 
'?' means it can be null at the beginning of the record creation.